.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
   - id: check-executables-have-shebangs
```

Comment:
Ah this is what is causing your PR to have more red, I'll move this to another PR and clean up there so its cleaner here.

Reply:
Fixed as of #431

Reply:
Thanks!